node-server/src/database/querier.js

```diff
@@ -0,0 +1,51 @@
+const sqlite3 = require('sqlite3').verbose()
+const logger  = require('../../../common/utils/logger')
+
+let db
+
+/**
+ * Opens or creates local sqlite3 database
+ */
+const initiateDatabase = async () => {
+  db = new sqlite3.Database('./dsatter.db', (err) => {
```

Comment:
Just a comment, especially since I don't fully understand how the DB-stuff works yet:  Maybe specify the path of the db-file in the module common/config or be passed as an argument? That way it might be easier to use a different DB for testing/dev/production depending on the current environment?